grossmont/computer-science-and-cybersecurity-organization/sections/header/header.css

```diff
@@ -63,32 +59,34 @@ body {
   /* color: #5CE1E6 */
 }
 
-.clubNameLower {
-  font-weight: lighter;
-  font-family: 'Sen', sans-serif;
-  font-size: 20px;
-  margin: 0;
-  color: #D9D9D9
+#clubLogo{
+  transform: scale(65%);
 }
```

Comment:
When I replaced the previous logo with this new one that has transparent background (alpha), It took up the entire height of the header, there I used this css property to scale it down a percentage relative to its parent size (the header bar).